### ISSUE TITLE
Feedback :) : install, UI, export CSV

### ISSUE THREAD

**Issue by olivier5741** [Newesissrl/payload-exportcollections-plugin#58]
Hi there, 

I just installed your plugin. I had to add this to my config in order to make it work (fs and crypto for sure, don't know about the 2 other ones)

```
webpack: config => ({
  ...config,
  resolve: {
    ...config.resolve,
    fallback: {
      "fs": false,
      "os": false,
      "path": false,
      "crypto": false
    },
...
```

The buttons are a bit big I would say, might be interesting to have them on top right above search bar (same level as the title) ?

<img width="1417" alt="Screenshot 2024-04-09 at 13 09 14" src="https://github.com/Newesissrl/payload-exportcollections-plugin/assets/3897291/71250270-d695-4501-9162-9191526b6136">

My model is rather big so with all the relationships the CSV has so many columns (94 !!). Maybe having a configurable deepness might help (maybe a deepness where you only get what is showed in the datatable).

Thanks for the work :)

**Comment by olivier5741** [comment]
Would be cool to export according to filters as well ;)

**Comment by finkinfridom** [comment]
hey @olivier5741 , thanks for the feedback above.
1. I'll double-check the configuration (as you provide it) as I remember it was perfectly working (but maybe something changed on the way)
2. the current layout used (i.e. for the size of the buttons) is something we inherit from the payloadcms core UI. I just saw it exists a `BeforeList` property but generally speaking, it may be useful to have this placement to be configurable on the developer's end.
3. Configuring the list of fields was already something I had in mind but for our current scenario was not needed. I'll take a note of this.
4. Accessing the filter while exporting is a really interesting feature and taking note of this as well.

Let you know when something will change.

Thanks again for the useful feedback.

**Comment by finkinfridom** [comment]
hey @olivier5741 , just released a `0.1.0` version with the above changes.
